src/sentry/api/endpoints/notification_defaults.py

```diff
@@ -0,0 +1,48 @@
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.base import Endpoint, control_silo_endpoint
+from sentry.notifications.defaults import (
+    NOTIFICATION_SETTING_DEFAULTS,
+    NOTIFICATION_SETTINGS_ALL_SOMETIMES,
+)
+from sentry.notifications.types import (
+    NOTIFICATION_SETTING_OPTION_VALUES,
+    NOTIFICATION_SETTING_TYPES,
+)
+from sentry.types.integrations import EXTERNAL_PROVIDERS
+
+# create the data structure outside the endpoint
+provider_defaults = []
+for key, value in NOTIFICATION_SETTING_DEFAULTS.items():
```

Comment:
wanna put this in a method, that is set to a constant here? `PROVIDER_DEFAULTS = get_provider_defaults()` and `TYPE_DEFAULTS = ...`